recipe/bld.bat

```diff
@@ -4,6 +4,11 @@
 :: (set as env var so it's used by both meson and during build with g-ir-scanner)
 set "PKG_CONFIG_PATH=%LIBRARY_LIB%\pkgconfig;%LIBRARY_PREFIX%\share\pkgconfig;%BUILD_PREFIX%\Library\lib\pkgconfig"
 
+copy %LIBRARY_LIB%\libmp3lame.lib %LIBRARY_LIB%\mp3lame.lib
+copy %LIBRARY_LIB%\libx265.lib %LIBRARY_LIB%\x265.lib
+copy %LIBRARY_LIB%\libbz2.lib %LIBRARY_LIB%\bz2.lib
+copy %LIBRARY_LIB%\libxml2.lib %LIBRARY_LIB%\xml2.lib
```

Comment:
Please delete the files after the build process. Else they will end up in the built package.